pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -57,8 +57,23 @@ def gurobi_run(model_file, warmstart_file, soln_file, mipgap, options, suffixes)
             print("***The GUROBI solver plugin cannot extract solution suffix="+suffix)
             return
 
+    # Handle Gurobi environment options
+
+    env_class = {
+        None: Env,
+        'Env': Env,
+        'CloudEnv': Env.CloudEnv,
+        'ClientEnv': Env.ClientEnv,
+        'OtherEnv': Env.OtherEnv,
+    }[options.get('env_type', None)]
+
+    if 'env_options' in options:
+        env = env_class(*options['env_options'])
+    else:
+        env = None
```

Comment:
@DFuller134 I don't think we got this right the first time. Sorry. Should this be
```
env_class = {
    None: None,
    'Env': Env,
    'CloudEnv': Env.CloudEnv,
    'ClientEnv': Env.ClientEnv,
    'OtherEnv': Env.OtherEnv,
}[options.get('env_type', None)]

if env_class is not None:
    if 'env_options' in options:
        env = env_class(*options['env_options'])
    else:
        env = env_class()
else:
    env = None
```
?